### ISSUE TITLE
Clear master password entry upon switching app

### ISSUE THREAD

**Issue by Currrupted** [Kunzisoft/KeePassDX#321]

**Comment by J-Jamet** [comment]
A timer should be created that automatically removes the password from the field. I will look to implement the feature but it will not be for now. I already have many other things to do.

**Comment by Currrupted** [comment]
> A timer should be created that automatically removes the password from the field. I will look to implement the feature but it will not be for now. I already have many other things to do.

If you don't mind, I would be interested in looking for a solution for this. I don't have that much experience with the code of KeePassDX, but it should not be too difficult after some studying.

**Comment by J-Jamet** [comment]
Yes, of course, I encourage this kind of initiative. It allows to have a fresh look on the code. You should look in the PasswordActivity class.

**Comment by Currrupted** [comment]
> 
> 
> Yes, of course, I encourage this kind of initiative. It allows to have a fresh look on the code. You should look in the PasswordActivity class.

I will look into it soon, sorry for the inactivity. Hopefully I will have it finished by the end of the year.
